src/app/shared/components/appointment-modal/appointment-modal.component.ts

```diff
@@ -50,26 +51,30 @@ export class AppointmentModalComponent implements OnInit {
   public finalize() {
```

Comment:
```suggestion
  public finalize(): void {
```

finalize ficou meio estranho... 
save() talvez?